xcodeproj/internal/xcschemes/xcschemes.bzl

```diff
@@ -1206,17 +1228,40 @@ def _autogeneration_config(scheme_name_exclude_patterns = None):
                         ".*somePattern.*",
                         "^AnotherPattern.*",
                     ],
-                )
+                ),
             )
             ```
 
+        test: Options to use for the test target.
+
+            Example:
+
+            ```starlark
+            xcodeproj(
+                ...
+                scheme_autogeneration_config = xcschemes.autogeneration_config(
+                    test = xcschemes.test(
```

Comment:
Since we don't/won't support the full values of `xcschemes.test` (e.g. `test_targets`), we should have a different struct here. I can see value in a "test"-like struct since most of the values we could set globally. And instead of just saying that some fields aren't respected, a new struct is better so that we can add unique values later as well.

I propose `xcschemes.autogeneration.test`. `autogeneration` would be a struct that has `test = _autogeneration_test` as it's sole value. `_autogeneration_test` would be `_test`, but with just `options` as a argument.